model/textparse/protobufparse_test.go

```diff
@@ -26,8 +25,9 @@ import (
 	"github.com/prometheus/prometheus/model/exemplar"
 	"github.com/prometheus/prometheus/model/histogram"
 	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/util/testutil"
 
-	dto "github.com/prometheus/prometheus/prompb/io/prometheus/client"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
This needs to stay the same as before. (Probably changed by some auto-import tool? Or maybe the client_model version is used somewhere else? We only want to use the prometheus/prompb version in Prometheus's own code.)